src/rpc/blockchain.cpp

```diff
@@ -551,10 +565,14 @@ static RPCHelpMan getblockheader()
     if (!request.params[1].isNull())
         fVerbose = request.params[1].get_bool();
 
+    const auto count{request.params[2].isNull() ? 0 : request.params[2].getInt<int>()};
```

Comment:
This change has been implemented as follows:
```
bool returnArray = !request.params[2].isNull();
```
The negation operator `!` is used because if `count != null`, then an array is returned.